tests/restarting/from_7.0.0/UpgradeAndBackupRestore-2.toml

```diff
@@ -0,0 +1,69 @@
+[[test]]
+testTitle = 'SecondCycleTest'
+simBackupAgents = 'BackupToFile'
+clearAfterTest=false
+runConsistencyCheck=false
+
+    [[test.workload]]
+    testName = 'Cycle'
+    nodeCount = 30000
+    transactionsPerSecond = 2500.0
+    testDuration = 30.0
+    expectedRate = 0
+    keyPrefix = 'AfterRestart'
+    
+    [[test.workload]]
+    testName = 'Cycle'
+    nodeCount = 30000
+    transactionsPerSecond = 2500.0
+    testDuration = 30.0
+    expectedRate = 0
+    keyPrefix = 'BeforeRestart'
```

Comment:
I don't quite understand what's going on here. Is the `BeforeRestart` prefix of cycles meant to be here? In the first place, since `clearAfterTest=false` was set all the way through, wouldn't the `SaveDatabase` portion of `UpgradeAndBackupRestore-1.toml` already save the state of the cycle test run previously? In which case the backup restore is kind of redundant

Reply:
The `BeforeRestart`-prefix cycle and `AfterRestart`-prefix cycles are both backed up, since the backup is ongoing through the upgrade. The `RestoreBackup` workload clears these keys before restoring, so its restore is not redundant.

Reply:
Ok, I understand now. I believe there are still two sets of the `BeforeRestart`-prefix cycle run, once in `UpgradeAndBackupRestore-1.toml`, and once in `UpgradeAndBackupRestore-2.toml`. If the backup is ongoing through the upgrade, is it intended to restore  the `BeforeRestart`-prefix cycle twice?

Reply:
Oh, I see. Yeah, this should be removed, we only want to write to the `AfterRestart` prefix in the second test file. I'll make this change.